ios/Classes/TiBarcodeViewController.m

```diff
@@ -54,8 +54,12 @@ - (void)viewWillAppear:(BOOL)animated
 {
   [super viewWillAppear:animated];
 
+  [_overlayView updateViewsWithFrame:[UIScreen mainScreen].bounds];
   [[self view] addSubview:_overlayView];
   [[self view] bringSubviewToFront:_overlayView];
+#if HAS_AVFF
```

Comment:
This is a preprocessor macro, so it will be compiled into the module, making it depending on our settings. Don't we always use this?

Reply:
This macro is defined in TiBarcodeViewController.h of this module as -
#if !TARGET_IPHONE_SIMULATOR
#define HAS_AVFF 1
#endif

to ensure that barcode scanner runs only when it is device. In case of simulator it will show the overlay view. Previously also the same stuff was used. Thanks!

Reply:
Right, but once we compile the module, it will either be defined or not. For native apps, it will be added or not based on the deployment, but since modules are precompiled, it uses one or another. Do you know what I mean?

Reply:
@vijaysingh-axway ping

Reply:
@hansemannn I tested the app using the module on simulator and device both. It is working fine and as expected. Thanks!